lighthouse-core/computed/critical-request-chains.js

```diff
@@ -28,6 +28,12 @@ class CriticalRequestChains {
       return false;
     }
 
+    // Whenever a request is a redirect not all props are filled in correctly (resourceType is undefined)
+    // so we loop until we find the final request so we use the correct data #6675
```

Comment:
nit: `https://github.com/GoogleChrome/lighthouse/issues/6675` so it's clickable :)